toolchain/lex/lex_helpers.h

```diff
@@ -2,8 +2,8 @@
 // Exceptions. See /LICENSE for license information.
```

Comment:
Would it make sense to rename this file to just "helpers.h"?

I'm happy for that to be done in a separate PR (if at all).

Reply:
Double-checked that "helpers.h" is still preferred with "test_helpers.h" here too. Renamed.